### PR TITLE
Update mask requirement

### PR DESCRIPTION
**Pull request by balinterdi**

### CODE REVIEW

File: corona-safety/index.html [hunk 1]
```diff
@@ -12,8 +12,8 @@ <h2 class="pre-heading__text text-align">The Coronavirus & people’s safety at
     </p>
     <ul class="content-text">
       <li>Attendees will need to be either fully vaccinated, recovered, or tested (PCR test, no older than 72h or Antigen test, no older than 48h) and be able to prove that via the <a href="https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate" class="content-link">EU’s Digital COVID Certificate</a>. We will take photos of all attendees’ certificate barcodes when entering the venue on both days.</li>
-      <li>Everyone will need to wear masks indoors at all times.</li>
       <li>Seating will be set up in compliance with current social distancing regulations.</li>
+      <li>Attendees should wear masks while moving around indoors but are not required to do so while seated or outdoors.</li>
```
Comment (marcoow) [inline corona-safety/index.html:16]: I assume

```suggestion
      <li>Attendees must wear masks while moving around indoors but are not required to do so while seated or outdoors.</li>
```

?

Comment (jkintscher) [inline corona-safety/index.html:16]:
```suggestion
      <li>Attendees will need to wear masks while moving around indoors, but are not required to do so while seated or outdoors on the premises.</li>
```

**Review by jkintscher — APPROVED** [review]
“should” sounds too lenient. These are mandatory rules.

**Review by marcoow — APPROVED** [review]

**Comment by balinterdi** [comment]
To my understanding, informally they are  interchangeable, but you're both right, this is not the context to be informal in.

I committed Joschka's suggestion.

**Pull request merged**